libraries/stdlib/src/kotlin/util/Lazy.kt

```diff
@@ -73,19 +73,19 @@ internal object UNINITIALIZED_VALUE
 // internal to be called from lazy in JS
 internal class UnsafeLazyImpl<out T>(initializer: () -> T) : Lazy<T>, Serializable {
     private var initializer: (() -> T)? = initializer
-    private var _value: Any? = UNINITIALIZED_VALUE
+    private var _value: T? = null
```

Comment:
can try this
private object UninitializedValue
private var _value: Any? = UninitializedValue
  @Suppress("UNCHECKED_CAST")
    override val value: T
        @MainThread
        get() {
            if (_value === UninitializedValue) {
                _value = initializer!!()
                initializer = null
            }
            return _value as T
        }